docs/static/examples/BottomSheet.js

```diff
@@ -71,12 +73,18 @@ const sheetStyles = StyleSheet.create({
 });
 
 export default function App() {
+  const { colorScheme } = useColorScheme();
   const isOpen = useSharedValue(false);
 
   const toggleSheet = () => {
     isOpen.value = !isOpen.value;
   };
 
+  const contentStyle = useAnimatedStyle(() => ({
+    color: colorScheme === 'light' ? '#001a72' : '#f8f9ff',
+    textDecorationColor: colorScheme === 'light' ? '#001a72' : '#f8f9ff',
+  }));
```

Comment:
I don't see any shared value inside `useAnimatedStyle`. Perhaps we can simply use a plain JavaScript object 🤔 What are your thoughts on that?

Reply:
It's actually good idea (like proper way to do this), i was thinking about it, but was not sure - thanks!